csharp/ql/src/experimental/Security Features/CWE-759/HashWithoutSalt.cs

```diff
@@ -0,0 +1,74 @@
+// semmle-extractor-options: /r:System.Security.Cryptography.Primitives.dll /r:System.Security.Cryptography.Csp.dll /r:System.Security.Cryptography.Algorithms.dll
```

Comment:
Please move this test file, along with the `.expected` file to a new `Security Features/CWE-759` folder under https://github.com/github/codeql/tree/main/csharp/ql/test/experimental. You will also need to add a `.qlref` file.

Reply:
Moved the test files and added a <code>.qlref</code> file